### PR TITLE
chore: update readme and some packages

### PR DESCRIPTION
**Pull request by oguhpereira**

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
   "main": "./main.js",
```
Comment (dubisdev) [inline app/package.json:6]: Version must be also updated here

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "cerebro",
   "productName": "cerebro",
-  "version": "0.6.0",
+  "version": "0.7.0",
```
Comment (dubisdev) [inline package.json:4]: Next version should be 0.7.1

**Review by dubisdev — CHANGES_REQUESTED** [review]
Version must be updated in both `package.json` and `app/package.json`

I think should be v0.7.1

File: package.json (hunk 2 above)
Comment (oguhpereira) [reply package.json]: Done https://github.com/cerebroapp/cerebro/pull/594/commits/ed45724c03a0dc2e0b368f99761ceee5ce901d7e

File: app/package.json (hunk 1 above)
Comment (oguhpereira) [reply app/package.json:6]: Done https://github.com/cerebroapp/cerebro/pull/594/commits/ed45724c03a0dc2e0b368f99761ceee5ce901d7e

**Review by dubisdev — APPROVED** [review]
This should work

**Pull request merged**